src/storage/mongodb/MongoDBStorage.ts

```diff
@@ -293,6 +296,9 @@ export default class MongoDBStorage {
           await LockingManager.release(createDatabaseLock);
         }
       }
+
+      // Detect changed Config Crypto Key per tenant
+      await Cypher.detectConfigurationKey(tenantId);
```

Comment:
I think it's gonna be removed at some point. Could you please add a FIXME or TODO comment stating its removal?

Reply:
I can remove it right now, because I'm doing it in the Scheduler Task